src/WinGetUtil/Exports.cpp

```diff
@@ -173,16 +173,18 @@ extern "C"
     CATCH_RETURN()
 
     WINGET_UTIL_API WinGetValidateManifest(
-        WINGET_STRING manifestPath,
+        WINGET_STRING inputPath,
         BOOL* succeeded,
-        WINGET_STRING_OUT* message) try
+        WINGET_STRING_OUT* message,
+        WINGET_STRING mergedManifestPath,
+        BOOL isPartialManifest) try

```

Comment:
>isPartialManifest [](start = 13, length = 17)

We need to know the expected type, or return it, or the service code will need to parse the manifest enough to answer the question of "Does this file name pattern match the type in the manifest?".

Rather than a BOOL here, we should take in an enum with values for single file, split, sub-component of a split.  It could even be an in-out where we also let them give us "don't know" and we give them back the answer.